src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './GoodsList.scss';
+
+type Props = {
+  goods: string[],
+};
+
+export const GoodsList: React.FC<Props> = React.memo(
```

Comment:
Using memo seems to be a good solution but all you can do in parent component is pushing the start button to render the child.